crates/fuzzing/src/generators/table_ops.rs

```diff
@@ -132,17 +157,21 @@ mod tests {
 
         let expected = r#"
 (module
-  (import "" "gc" (func))
-  (table $table 10 externref)
-  (func (export "run") (param externref externref)
-    (call 0)
-    (drop (table.get $table (i32.const 0)))
-    (table.set $table (i32.const 1) (local.get 2))
-    (table.set $table (i32.const 3) (table.get $table (i32.const 4)))
-  )
-)
+  (type (;0;) (func (param externref externref)))
+  (import "" "gc" (func (;0;) (type 0)))
+  (func (;1;) (type 0) (param externref externref)
+    call 0
+    drop
+    table.get 0
+    table.set 1
+    local.get 2
+    table.set 3
+    table.get 4)
+  (table (;0;) 10 externref)
+  (export "run" (func 0)))
```

Comment:
Here is the fully elaborated/desugared WAT that we should expect:

```suggestion
  (type (;0;) (func))
  (type (;1;) (func (param externref externref)))
  (import "" "gc" (func (;0;) (type 0)))
  (func (;1;) (type 1) (param externref externref)
    call 0
    table.get 0
    drop
    local.get 2
    table.set 1
    table.get 4
    table.set 3)
  (table (;0;) 10 externref)
  (export "run" (func 1)))
```

Reply:
In the function body I needed to add the table index to your expected output ie. `table.get 0`